src/ngraph_encapsulate_op.cc

```diff
@@ -74,6 +74,10 @@ class NGraphEncapsulateOp : public OpKernel {
     my_instance_id = s_instance_count;
     s_instance_count++;
 
+    std::ostringstream oss;
+    oss << "Encapsulate_" << my_instance_id << ": " << name();
+    Event event(oss.str().c_str(), name().c_str());
+
```

Comment:
Do we need `my_instance_id`? `name()` already provides a unique name and also embeds the encapsulate ID (`m_ngraph_cluster`) information in the name

Reply:
The instance id is useful for other correlations. So will keep it for now.